mage/env.go

```diff
@@ -34,7 +31,7 @@ func getAmbientEnvironment() Environment {
 	default:
 		registry := os.Getenv("PORTER_OPERATOR_REGISTRY")
 		if registry == "" {
-			mgx.Must(errors.New("environment variable PORTER_OPERATOR_REGISTRY must be set to push to a custom registry"))
+			registry = "localhost:5000"
```

Comment:
What was the reason for add the default value here?  Does the match the porter magefile pattern?